crates/pep508-rs/src/verbatim_url.rs

```diff
@@ -198,21 +181,12 @@ fn expand_env_vars(s: &str, escape: Escape) -> Cow<'_, str> {
     RE.replace_all(s, |caps: &regex::Captures<'_>| {
         let name = caps.name("name").unwrap().as_str();
         std::env::var(name).unwrap_or_else(|_| match name {
-            // Ensure that the variable is URL-escaped, if necessary.
-            "PROJECT_ROOT" => match escape {
-                Escape::Url => PROJECT_ROOT_FRAGMENT.replace(' ', "%20"),
```

Comment:
I believe this isn't necessary. If the root is in a `file://`, we already strip that as a special case and parse this as a path, so spaces are fine.